AK Zeus/app/src/main/java/ayds/zeus/songinfo/moredetails/fulllogic/OtherInfoWindow.kt

```diff
@@ -0,0 +1,122 @@
+package ayds.zeus.songinfo.moredetails.fulllogic
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.text.HtmlCompat
+import ayds.zeus.songinfo.R
+import com.google.gson.Gson
+import com.google.gson.JsonElement
+import com.google.gson.JsonObject
+import com.squareup.picasso.Picasso
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.scalars.ScalarsConverterFactory
+import java.util.*
+
+class OtherInfoWindow : AppCompatActivity() {
```

Comment:
En android es buena práctica agregar el sufijo Activity
```suggestion
class OtherInfoActivity : AppCompatActivity() {
```